### ISSUE TITLE
Visualization Graphviz /dotgraph diagram with "Show hierarchy" (types in domain, and objects in problem) fail since version V2.25.2 included

### ISSUE THREAD

**Issue by AKEOPLUS-boris-bocquet** [jan-dolejsi/vscode-pddl#150]
Trying this : 
```
	(:types
		workspace - location
		arm truck goods part container equipment - locatable
		paste-applicator - equipment
		location locatable
	)
```
works as expected when clicking on "show hierarchy" in version bellow V2.25.2.
But it shows nothing (open a tab window with no diagram) starting at version V2.25.2.

My setup : 
Windows 10
Docker with PlanUtils (50ca0f5915596d2e27d4c0eb820483415e09c1337d49eb316f3683b50faee581)
WSL2
Visual Studio Code 1.76.2
Electron 19.1.11
Chromium 102.0.5005.196
NodeJs 16.14.2

**Comment by jan-dolejsi** [comment]
My bad. Deployment configuration issue. I believe it is fixed now. Can you please  confirm?

**Comment by AKEOPLUS-boris-bocquet** [comment]
OK this problem looks corrected at least starting at version V2.25.7 . Thanks Jan.
